src/Lean/Server/FileWorker.lean

```diff
@@ -424,8 +424,8 @@ def setupImports
 
   return .ok {
     mainModuleName := doc.mod
-    isModule := header.isModule
-    imports := header.imports
+    isModule := strictOr setup.isModule header.isModule
```

Comment:
This is always equivalent to `header.isModule`, right?

Reply:
Currently, from the Lake side, yes.